docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/snowplow-tag-for-gtm-ss/snowplow-tag-configuration/index.md

```diff
@@ -38,52 +38,42 @@ Allows the expiration time of the cookie to be altered. This value is in seconds
 
 #### Secure
 
-Overwrites the Secure flag on the cookie. Will be `true` if enabled, or `false` if disabled. Setting this to `false` and `SameSite` to `None` will prevent browsers from being able to store the cookie.
+Overwrites the Secure flag on the cookie. Will be `true` if enabled (default), or `false` if disabled. Setting this to `false` and `SameSite` to `None` will prevent browsers from being able to store the cookie.
 
-## Advanced Event Settings
-
-### Define events to be sent as Snowplow Self-Describing Events
-
-Enable this to allow custom self-describing event definitions.
-
-#### Event Name to Schema
-
-A table of the events (event names and corresponding schemas) to be tracked as custom self-describing Snowplow events.
+### Cookie Headers
 
-Add events which you would like to capture and convert into Self Describing Events. The `Event Name` should equal the Clients `event_name` property, if this is found when the Snowplow Tag fires, this tag will create a Self Describing Event using the specified schema.
+![cookie headers](images/cookie_headers.png)
 
-#### Event Definitions
+#### Forward Cookies in the request headers
 
-A table of definitions for self-describing data properties. Each row maps a single data property of a custom self-describing Snowplow event to its value.
+This option allows you to forward selected cookies in the request headers and it is disabled by default. Enabling it reveals the "Cookies to forward" table.
 
-For each Event, you can also read properties off the client event object and add them as properties to the Self Describing Event.
+#### Cookies to forward
 
-![](images/sdjsetup.png)
+Using this table allows you to select the cookies to forward. Its columns are:
 
-### Send selected events as Snowplow Structured Events
+- **Cookie Name**: The name of the cookie to forward
+- **Decode**: Whether to want to decode the cookie value(s) before forwarding (defaults to `No`)
 
-Enable this to allow setting custom Structured Events.
-
-#### Event name(s) selected
-
-Add the event names (in separate lines) to be tracked as custom structured Snowplow events.
+## Advanced Event Settings
 
-### Base64 encoding
+![advanced event settings overview](images/advanced_event_settings_overview.png)
 
-Whether to encode the custom self-describing event data in base64.
+This section allows you to:
 
-### Platform identifier
+- Specify events to be tracked as Structured events
+- Configure Self-describing event definitions
+- Specify Context entities to be attached to events
+- Customize additional event settings
 
-When a platform is not specified on the event, this value will be used.
+You can find out more about all the available configuration options in this [dedicated page here](/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/snowplow-tag-for-gtm-ss/snowplow-tag-configuration/advanced-event-settings/index.md).
```

Comment:
I think this single link here is fine. However, I've always been told its good to add some context to links rather than saying "click here" or similar from an SEO point of view (no idea if crawlers actually use this but it can't hurt).

```suggestion
You can find out more about all the available configuration options in the [Advanced Event Settings page](/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/snowplow-tag-for-gtm-ss/snowplow-tag-configuration/advanced-event-settings/index.md).
```